<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trade</class>
 <widget class="QWidget" name="Trade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trade</string>
  </property>
  <widget class="QFrame" name="frame_window">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>390</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame[objectName=&quot;frame_window&quot;]
{
	background-color: rgb(250, 250, 250);
	border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>390</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;
color:rgb(20,20,20);
padding-bottom:1px;
border-top-left-radius:14px;
border-top-right-radius:14px;
background-color: rgb(110, 205, 255)</string>
    </property>
    <property name="text">
     <string>交易窗口</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	border-top-right-radius:14px;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(255, 0, 0);
}

</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>对方账户：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_account">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>65</y>
      <width>226</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_money">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>125</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hint_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>125</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px;</string>
    </property>
    <property name="text">
     <string>交易金额：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>115</y>
      <width>31</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:30px</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Trade</receiver>
   <slot>btn_close_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_close_click()</slot>
 </slots>
</ui>
